### PR TITLE
[4.0]com_media missing download lang string

### PR DESCRIPTION
**Pull request by bahl24**
Signed-off-by: Nitish Bahl <nitishbahl24@gmail.com>

Pull Request for Issue - Download string is used in many .vue files, but translation missing in lang files(media manager) .
I am not sure if this is duplicate or not, kindly check

### CODE REVIEW

**Comment by bahl24** [comment]
@infograf768 Can you test this?

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 24b3611770a0ca64f8541429db8b009ec4737aa4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23967">issues.joomla.org/tracker/joomla-cms/23967</a>.</sub>

**Comment by bahl24** [comment]
@Quy @HLeithner Kindly review

**Review by dneukirchen — APPROVED** [review]

File: administrator/language/en-GB/en-GB.com_media.ini [hunk 1]
```diff
@@ -6,6 +6,7 @@
 COM_MEDIA="Media"
 COM_MEDIA_ACTIN_RENAME="Rename item"
 COM_MEDIA_ACTION_DELETE="Delete item"
+COM_MEDIA_ACTION_DOWNLOAD="Download"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_media.ini]: Should it be `Download item`?

Comment (bahl24) [reply administrator/language/en-GB/en-GB.com_media.ini]: screenshot from 2019-02-27 01-02-33
I think dowmload is fine

Comment (Quy) [reply administrator/language/en-GB/en-GB.com_media.ini]: To be consistent with the other related strings.

Comment (bahl24) [reply administrator/language/en-GB/en-GB.com_media.ini]: Ok

**Comment by bahl24** [comment]
@Quy I have modified it, kindly test

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a5188f88c9f0a69745a6e12f565cc1e5394df4e2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23967">issues.joomla.org/tracker/joomla-cms/23967</a>.</sub>

**Comment by bahl24** [comment]
@Quy @infograf768 Any idea why drone is failing?

**Comment by bahl24** [comment]
@Quy  RTC?

**Comment by ghost** [comment]
~~@bahl24 can you please merge the latest changes from 4.0-dev?~~

@infograf768 @Quy can you please retest?

**Comment by bahl24** [comment]
Is there any problem?

**Comment by ghost** [comment]
screen shot 2019-03-03 at 10 18 06

**Comment by infograf768** [comment]
There are some problems with drone. Should be solved soon.

**Comment by brianteeman** [comment]
@franz-wohlkoenig  a branch being out of date is not a problem - unless there are merge conflicts

**Comment by ghost** [comment]
@brianteeman thanks for remembering me.

**Comment by bahl24** [comment]
Kindly test this

**Comment by Quy** [comment]
@infograf768 Are you ok with the language change?

**Comment by infograf768** [comment]
> @infograf768 Are you ok with the language change?

yep

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23967">issues.joomla.org/tracker/joomla-cms/23967</a>.</sub>

**Comment by infograf768** [comment]
As this would conflict with
https://github.com/joomla/joomla-cms/pull/23951
It is better to first correct that one and then merge this one.

**Comment by infograf768** [comment]
thanks

**Pull request merged**